packages/user_packages/iotcc_mqtt_edge_system_stats.py

```diff
@@ -65,7 +65,14 @@ def read_cpu_utilization(sample_duration_sec=1):
 
 
 def read_disk_usage_stats():
```

Comment:
Can you also add an example, where say after publishing 3-4 values, an exception is thrown, to show it's collection getting stopped?